snippets/base/templates/base/includes/snippet.js

```diff
@@ -661,7 +661,9 @@ function popFromBlockList(snippetID) {
 
 function addToBlockList(snippetID) {
     var blockList = getBlockList();
-    snippetID = parseInt(snippetID, 10);
```

Comment:
`snippetID` used to be a string (now it's an int) and I parsed it as int so it's faster/better to store and lookup.

With this change we still keep snippetID as int and we also allow strings for campaigns.